### PR TITLE
Migrate from buf's remote generation alpha

### PR DESCRIPTION
**Pull request by dmah42**
more build issues... still hacking away

### CODE REVIEW

**Comment by dmah42** [comment]
> check deny and check docs are timeouts. so i think this is ready to go.

timeouts resolved by extending them. check deny is still freaking out but i'm not sure how big a deal that is :)

**Comment by dmah42** [comment]
seems like somehow youki/libcgroups lost its LICENSE file between v0.0.4 and v0.0.5...

File: auraescript/src/cells.rs [hunk 1]
```diff
@@ -28,4 +28,6 @@
  *                                                                            *
 \* -------------------------------------------------------------------------- */
 
+use deno_core::Op;
```
Comment (future-highway) [inline auraescript/src/cells.rs:31]: Nit: preferably this would be generated by the macro

**Review by future-highway — APPROVED** [review]
I was able to build, install and run aurae and auraescript (`cells.ts` example) successfully.

Also looked at the check deny run and agree that the errors don't look like a big deal.

**Pull request merged**